packages/caip/src/constants.ts

```diff
@@ -67,7 +67,7 @@ export const CHAIN_REFERENCE = {
   DogecoinMainnet: '00000000001a91e3dace36e2be3bf030',
   LitecoinMainnet: '12a765e31ffd4059bada1e25190f6e98',
   CosmosHubMainnet: 'cosmoshub-4',
-  ThorchainMainnet: 'thorchain-mainnet-v1',
```

Comment:
We might want to keep `thorchain-mainnet-v1` here for backwards compatibility.

E.g. if I run this branch with an existing cache:

<img width="1182" alt="Screenshot 2024-09-09 at 1 47 40 PM" src="https://github.com/user-attachments/assets/4bc24b50-e51d-4e68-8bfb-9047f33a4790">

Reply:
I had migrations, but added them for assets, txhistory, portfolio and opportunities so this ideally shouldn't be an issue.